docs/pages/build/building-from-github.mdx

```diff
@@ -214,10 +214,41 @@ On your project's GitHub settings page in the Expo dashboard, you can click the
 a build trigger row to disable, edit, or delete the trigger.
 
 <ImageSpotlight
-  alt="Manage GitHub build trigger with the options button"
-  src="/static/images/eas-build/build-from-github/managing-build-triggers.png"
+  alt="The options button on the build trigger row"
+  src="/static/images/eas-build/build-from-github/edit-trigger-option.png"
 />
 
+You can also run a GitHub build with the parameters from the trigger manually. This will not count towards your automatic build trigger record.
+
+#### Automatic Store Submission with EAS Submit
+
+Once your build completes, you can automatically submit your app to the digital stores using EAS Submit. This feature streamlines the process, reducing the manual steps required to publish your app.
+
+To enable automatic submission, you need to configure your build triggers to include submission as part of the build process. Here's how you can set it up:
+
+- Navigate to your project's GitHub settings page on the Expo dashboard.
+- Find the build trigger you want to modify, and click the options button.
+- Select "Edit trigger" and in the dialog that appears, check the option "Submit to store after
```

Comment:
```suggestion
- Select **Edit trigger** and in the dialog that appears, check the option **Submit to store after build**.
```

- We use bold names to represent options/settings clicked in Expo dashboard. Let's do the same here for consistency.